be/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
add .idea or what ever @Orzelius added to fe to ignore intellij editor settings files

Reply:
I added it to the gitignore in the root folder, so nwed to re-add it
https://github.com/TPT-Loane/TPT-Loane/blob/508472ac2fd23c3a3beac7cefe24a1b5e4dc92d7/.gitignore#L10

Reply:
oh right its still one repo.